admin-dev/themes/new-theme/js/pages/order/view/order-payments-refresher.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+import Router from '@components/router';
+import OrderViewPageMap from '@pages/order/OrderViewPageMap';
+
+const {$} = window;
+
+export default class OrderPaymentsRefresher {
```

Comment:
I'm just a bit disturbed by this naming (in route and urls as well) It feels like you are updating the list of payments but really it only updates the Payment Alert
I don't block this since it already has two approvals but I really think naming could be improved, and since QA is quite busy for now you should have time to fix it

The overall PR is fine besides this naming issue